services/app-api/src/emailer/formatters/formatEmailAddresses.test.ts

```diff
@@ -99,9 +99,15 @@ describe('pruneDuplicateEmails', () => {
             ['"Foo Bar 1" foo@bar.com', '"Foo Bar 2" <foo@bar.com>'],
             ['"Foo Bar 1" foo@bar.com', '"Foo Bar 2" <foo@bar.com>'],
         ],
+        // simple email addresses that are duplicates due to casing are pruned and lowercase preferred
         [
-            ['foo@bar.com', 'foo@nothing.com', 'bar@foo.com', 'bar@foo.com'],
-            ['foo@bar.com', 'foo@nothing.com', 'bar@foo.com'],
+            ['Foo@bar.com', 'Fo.b-r@bar.com', 'fo.b-r@bar.com', 'foo@bar.com'],
+            ['Foo@bar.com', 'Fo.b-r@bar.com'],
```

Comment:
Could we also add tests with different casing in the email and alias? Just want to make sure we test a range of possibilities.

Reply:
You mean something like this?
```javascript
 ['"foo bar 1" FOO@BAR.COM', '"Foo Bar 2" <foo@bar.com>'],
 ['"foo bar 1" FOO@BAR.COM', '"Foo Bar 2" <foo@bar.com>'],
```

Reply:
Yes. I think you could edit the first set of emails in `sampleEmailLists` to include casing in alias emails.

Reply:
The idea I had in mind was to have a set of emails with all the scenarios so we can rigorously test the function.